packages/runner/tsconfig.json

```diff
@@ -51,8 +51,9 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "noErrorTruncation": true,
-    "experimentalDecorators": true
+    "experimentalDecorators": true,
     // "noResolve": true
+    "resolveJsonModule": true
```

Comment:
was `json` added anywhere and is this still relevant?

Reply:
Yeah, we need it for the set attribute patch: https://github.com/cypress-io/cypress/blob/matth/cy-origin-async-attach/packages/runner/injection/patches/setAttribute.ts#L1